.gitignore

```diff
@@ -1,2 +1,3 @@
+/dist/
```

Comment:
Also, when this is ready to go, we should try publishing it to npm under a next or beta tag.